docs/pages/versions/unversioned/sdk/updates.mdx

```diff
@@ -45,11 +45,153 @@ The remote service must implement the [Expo Updates protocol](/technical-specs/e
+| `updates.enabled`                                           | `true`   | <NoIcon />  | `EXUpdatesEnabled`                | `expo.modules.updates.ENABLED`                                   | `enabled`                |
+| `updates.url`                                               | (none)   | <YesIcon /> | `EXUpdatesURL`                    | `expo.modules.updates.EXPO_UPDATE_URL`                           | `updateUrl`              |
+| `updates.requestHeaders`                                    | (none)   | <NoIcon />  | `EXUpdatesRequestHeaders`         | `expo.modules.updates.UPDATES_CONFIGURATION_REQUEST_HEADERS_KEY` | `requestHeaders`         |
+| `runtimeVersion`                                            | (none)   | <YesIcon /> | `EXUpdatesRuntimeVersion`         | `expo.modules.updates.EXPO_RUNTIME_VERSION`                      | `runtimeVersion`         |
+| `updates.checkAutomatically`                                | `ALWAYS` | <NoIcon />  | `EXUpdatesCheckOnLaunch`          | `expo.modules.updates.EXPO_UPDATES_CHECK_ON_LAUNCH`              | `checkOnLaunch`          |
+| `updates.fallbackToCacheTimeout`                            | `0`      | <NoIcon />  | `EXUpdatesLaunchWaitMs`           | `expo.modules.updates.EXPO_UPDATES_LAUNCH_WAIT_MS`               | `launchWaitMs`           |
+| `updates.useEmbeddedUpdate`                                 | `true`   | <NoIcon />  | `EXUpdatesHasEmbeddedUpdate`      | `expo.modules.updates.HAS_EMBEDDED_UPDATE`                       | `hasEmbeddedUpdate`      |
+| `updates.codeSigningCertificate`                            | (none)   | <NoIcon />  | `EXUpdatesCodeSigningCertificate` | `expo.modules.updates.CODE_SIGNING_CERTIFICATE`                  | `codeSigningCertificate` |
+| `updates.codeSigningMetadata`                               | (none)   | <NoIcon />  | `EXUpdatesCodeSigningMetadata`    | `expo.modules.updates.CODE_SIGNING_METADATA`                     | `codeSigningMetadata`    |
+| `updates.assetPatternsToBeBundled`                          | (none)   | <NoIcon />  | N/A                               | N/A                                                              | N/A                      |
+
+The two core required configuration options are:
+
+- [`updates.url`](/versions/latest/config/app/#updates): the URL at which the library fetches remote updates
+- [`runtimeVersion`](/versions/latest/config/app/#runtimeversion): a [runtime version](#runtime-version)
+
+These are configured automatically when following the EAS Update [Get started](/eas-update/getting-started/) guide.
+
+#### Runtime version
+
+Each time you build a binary for your app it includes the native code present at the time of the build as well as native configuration, and this unique combination is represented by a unique string called a runtime version. A remote update targets one runtime version, indicating that only binaries with a matching runtime version can load the remote update.
```

Comment:
```suggestion
Each time you build a binary for your app it includes the native code present at the time of the build as well as native configuration, and this unique combination is represented by a string called a runtime version. A remote update targets one runtime version, indicating that only binaries with a matching runtime version can load the remote update.
```

In the first sentence, the string is already unique since the combination is unique, so we can avoid repetition.